### ISSUE TITLE
U2Net tf pb model works fine , but tflite model works wrong

### ISSUE THREAD

**Issue by AR-fan** [tensorflow/tensorflow#48823]
The project is from [pytorch model](https://github.com/xuebinqin/U-2-Net)  
I use [colab](https://colab.research.google.com/drive/10bPSzQL8-im6RdP8fXm2_VCcbETGvcNM?usp=sharing)
I put two result pics together.    

Only need to update my_tf_model.pb-20210429T083200Z-001.zip in colab.  
 
[my_tf_model.pb-20210429T083200Z-001.zip](https://github.com/tensorflow/tensorflow/files/6397925/my_tf_model.pb-20210429T083200Z-001.zip)  

[my_tf_model.zip](https://github.com/tensorflow/tensorflow/files/6397926/my_tf_model.zip)  
[my_model2.zip](https://github.com/tensorflow/tensorflow/files/6397932/my_model2.zip)

**Comment by abattery** [comment]
Please elaborate more on the expected results.

**Comment by AR-fan** [comment]
The expected result is like pb model's output.

And there is some [expected result](https://github.com/xuebinqin/U-2-Net).  

I have updated colab, thanks.

**Comment by abattery** [comment]
Could you try conversion without the model optimization?

**Comment by AR-fan** [comment]
When I comment this code:  
    ```converter.optimizations = [tf.lite.Optimize.DEFAULT]```  
It works fine, thank you very much!

**Comment by AR-fan** [comment]
I have another questions, pls:   
1.Don't use optimizations , time speed transforms from 800ms to 1700ms, is it normal? How to balance time and accuracy?   
2.The tflite model's input is [1,3,224,224], should I do some special operation  to use it in android? Shoudl I change the shape to [1, 224,  224, 3]? Result is not good in my android app.

**Comment by tilakrayal** [comment]
@AR-fan ,

Could you please confirm if the above mentioned issue is resolved. if yes, please feel free to move this issue to closed status and please submit a new issue from this link and fill in the template, so that we can track the issue there. Thanks!

**Comment by abattery** [comment]
Let's use this for the remaining model optimization toolkit issue.

**Comment by AR-fan** [comment]
Hello everyone , how is it going？

**Comment by abattery** [comment]
Could you upload a new issue for the left-over model optimization toolkit? Looks like this issue already is closed and it is hard to get attentions due to the long history.

**Comment by AR-fan** [comment]
OK, thank you!

**Comment by Georgeiva** [comment]
--
<string>1:947996152464:ios:458174e3f31524b50afe15</string> martes, 25 mayo 2021, 04:50a. m. -05:00 de tensorflow-butler[bot]  ***@***.*** :
